### DISCUSSION TITLE
Essentials Kit Creation

### DISCUSSION THREAD

**Discussion by Crackheadzoro** [EssentialsX/Essentials#5101] [Q&A] [answered]
Started a Pixelmon server in Aternos and I am having issues setting up a starter kit for the server. And every time I enter the Command for the kit, I get a error message stating "That kit is improperly defined. Contact an administrator". And I went to the kit.yml and it shows the items as null.

**Comment by pop4959** [comment, marked as answer]
Keep in mind that Essentials is not supported on Forge or hybrid server software designed to run alongside mods.

If you are not running in that environment, please share your Essentials dump output `/ess dump all`, the server's error log, and the specific command you were trying to run which didn't work.

**Reply by Crackheadzoro** [reply]
When I did /ess dumb all, i get this link https://essentialsx.net/dump.html?id=2eaeb652654e49c3ac530d6d01dd7042,
and I don't know what to do with it. As for the command i did /kit Starter and /kit balls.

**Reply by pop4959** [reply]
Mohist is not supported by Essentials. Please use a supported server software and it should work.

**Reply by Crackheadzoro** [reply]
okay thank you aprreciate it.
